node-hub-api-adapter/helm/hub-adapter/values.yaml

```diff
@@ -5,10 +5,10 @@ env:
     API_CLIENT_ID: hub-adapter
     IDP_REALM: flame
     IDP_URL: http://keycloak-service:8080
-    RESULTS_SERVICE_URL: http://results-service:5000
+    RESULTS_SERVICE_URL: http://node-result-service:8080
```

Comment:
Shouldn't this be `flame-node-result-service` or `{{ .Release.Name }}-result-service:8080` for that matter?

Reply:
In the `values.yaml` for individual components, you can set whatever you want since they should be overwritten using the parent `flame` helm chart. But it is a good idea to set this to an empty string so the `_helpers.tpl` will auto fill it, good catch